README.md

```diff
@@ -2,15 +2,16 @@
 
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** with your full name, so as to make your ReadMe more professional. Also, endeavor to add the name(s) of any other contributor.